### PR TITLE
cmake changes

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by tchaikov** [comment]
retest this please.

File: src/script/CMakeLists.txt [hunk 1]
```diff
@@ -5,3 +5,11 @@ if(WITH_SUBMAN)
     ${CMAKE_CURRENT_SOURCE_DIR}/subman
     DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/cron.hourly)
 endif()
+
+configure_file(
```
Comment (tchaikov) [inline src/script/CMakeLists.txt]: @markhpc may you will like this!

**Comment by alimaredia** [comment]
@mdw-at-linuxbox could you look over the changes in FindOpenLdap.cmake?

**Comment by wjwithagen** [comment]
@tchaikov 
Looks like this is a good reason to rebase. :)

**Pull request merged**